setup.py

```diff
@@ -45,7 +45,17 @@
     # Python versions (enforced)
     python_requires='>=3.0.0, <4',
     # deps installed by pip
-    install_requires=['eventemitter', 'asyncio', 'websockets', 'pylint', 'six', 'pyee', 'aiohttp'],
+    install_requires=[
+        'eventemitter==0.2.0',
+        'asyncio==3.4.3',
+        'websockets==7.0',
+        'pylint==2.3.0',
+        'pytest-asyncio==0.10.0',
+        'six==1.12.0',
+        'pyee==8.0.1',
+        'aiohttp==3.4.4',
+        'isort==4.3.21'
+    ],
```

Comment:
Hey there, I'm sorry, but just to say that this is a bad practice. As a library, this should only enforce the range (minimum and max) of versions of the dependencies it needs to run. Also, `pylint` and `pytest` are dev dependencies.

By doing what you just did you are enforcing users to use specific versions, and this might conflict with other dependencies. For example if another library requires `six==1.12.1`, this will make it fail, and even more with the latest versions of pip v21, where they make sure that no the dependencies match when two libraries depend in the same version.

Also, you just enforced me as a user to use `pylint==2.3.0`, so keep dev dependencies out of `setup.py`.

To manage this, just use a requirements.txt or a Pipenv, or Poetry, but `setup.py` really needs to define the range of supported dependencies!

Do you want me to open a new PR and revert this changes, or...?

Reply:
> By doing what you just did, you are enforcing users to use specific versions and this might conflict with other dependencies.

Or tell users to use an isolated virtualenv when using bitfinex-api-py.

Reply:
> > By doing what you just did, you are enforcing users to use specific versions and this might conflict with other dependencies.
> 
> Or tell users to use an isolated virtualenv when using bitfinex-api-py.

No, that's not true. Already using venvs here. If in the same venv you install another dependency that uses aiohttp or asyncio (for example any simple API framework), you are still forcing bitfinex-api-py's version.

Libraries such as this one only need to require the range of versions of their dependencies and allow users to install the version they want. Otherwise, do I need to wait for this library to update in order to update the pytest-asyncio version of my project?

In sum, this will just provoke conflicts nonstop if you leave it as is....

Reply:
`isort` is also a dev dependencies and should not be included in production.

Reply:
The same can be said for `pylint` as well.